src/encoded/schemas/experiment_series.json

```diff
@@ -0,0 +1,211 @@
+{
+    "title": "Experiment series",
+    "description": "Schema for submitting metadata for a group of experiments which are applying the same assay on the same type of biosample.",
+    "id": "/profiles/experiment_series.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "required": [ "related_datasets"],
+    "identifyingProperties": ["uuid", "accession", "aliases"],
+    "additionalProperties": false,
+    "mixinProperties": [
+        { "$ref": "mixins.json#/schema_version" },
+        { "$ref": "mixins.json#/uuid" },
+        { "$ref": "mixins.json#/accession" },
+        { "$ref": "mixins.json#/aliases" },
+        { "$ref": "mixins.json#/attribution" },
+        { "$ref": "mixins.json#/submitted" },
+        { "$ref": "mixins.json#/notes" },
+        { "$ref": "mixins.json#/submitter_comment" },
+        { "$ref": "series.json#/properties" }
+    ],
+    "dependencies": {
+        "status": {
+            "oneOf": [
+                {
+                    "required": ["date_released"],
+                    "properties": {
+                        "status": {
+                            "enum": ["released", "revoked"]
+                        }
+                    }
+                },
+                {
+                    "not": {
+                        "properties": {
+                            "status": {
+                                "enum": ["released", "revoked"]
+                            }
+                        }
+                    }
+                }
+            ]
+        }
+    },
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "accession": {
+            "accessionType": "SR"
+        },
+        "award": {
+            "default": "5854a1dc-4c50-44b3-ba58-0a6f4f399297",
```

Comment:
is this normally how we do this?

Reply:
No. But we do it in dependency: https://github.com/ENCODE-DCC/encoded/blob/dev/src/encoded/schemas/biosample.json#L50-L65
The reason is I think we can/should restrict it to just one value. But I could be wrong about it.

Reply:
@gabdank do we need to actually set the default award in the schema?...   I don't think we do.  It's not a strict requirement that we own them all, just that we will likely do so.

Reply:
I don't think we should set up a default award in the schema. The reason it is used in biosample - was to restrict model organism properties to non-human and not mouse organisms.